akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/StashManagement.scala

```diff
@@ -47,15 +42,18 @@ private[akka] trait StashManagement[C, E, S] {
 
     try buffer.stash(msg) catch {
       case e: StashOverflowException ⇒
-        setup.stashOverflowStrategy match {
-          case DiscardToDeadLetterStrategy ⇒
-            val noSenderBecauseAkkaTyped: ActorRef = untyped.ActorRef.noSender
-            context.system.deadLetters.tell(DeadLetter(msg, noSenderBecauseAkkaTyped, context.self.toUntyped))
-
-          case ReplyToStrategy(_) ⇒
-            throw new RuntimeException("ReplyToStrategy does not make sense at all in Akka Typed, since there is no sender()!")
-
-          case ThrowOverflowExceptionStrategy ⇒
+        setup.settings.stashOverflowStrategy match {
+          case StashOverflowStrategy.Drop ⇒
+            if (context.log.isWarningEnabled) {
+              val dropName = msg match {
+                case InternalProtocol.IncomingCommand(actual) ⇒
+                  actual.getClass.getName
```

Comment:
I'd put this on the same line as case or have line 51 go to the next line for format consistency joy.